tests/test_flask_util.py

```diff
@@ -272,36 +285,44 @@ def test_callback_view_errors(self):
                 'return_url': '/return_url'
             })
 
-            rv = c.get('/oauth2callback?state=%s&code=codez' % state)
-            self.assertEqual(rv.status_code, httplib.BAD_REQUEST)
+            response = client.get('/oauth2callback?state=%s&code=codez' % state)
+            self.assertEqual(response.status_code, httplib.BAD_REQUEST)
 
         # KeyError, no CSRF state.
-        with self.app.test_client() as c:
-            rv = c.get('/oauth2callback?state={}&code=codez')
-            self.assertEqual(rv.status_code, httplib.BAD_REQUEST)
+        with self.app.test_client() as client:
+            response = client.get('/oauth2callback?state={}&code=codez')
+            self.assertEqual(response.status_code, httplib.BAD_REQUEST)
 
         # Code exchange error
-        with self.app.test_client() as c:
-            with Http2Mock(status=500):
-                with c.session_transaction() as session:
-                    session['google_oauth2_csrf_token'] = 'tokenz'
-
-                state = json.dumps({
-                    'csrf_token': 'tokenz',
-                    'return_url': '/return_url'
-                })
+        with self.app.test_client() as client:
+            state = self._setup_callback_state(client)
 
-                rv = c.get('/oauth2callback?state=%s&code=codez' % state)
-                self.assertEqual(rv.status_code, httplib.BAD_REQUEST)
+            with Http2Mock(status=httplib.INTERNAL_SERVER_ERROR):
+                response = client.get(
+                    '/oauth2callback?state=%s&code=codez' % state)
+                self.assertEqual(response.status_code, httplib.BAD_REQUEST)
 
         # Invalid state json
-        with self.app.test_client() as c:
-            with c.session_transaction() as session:
+        with self.app.test_client() as client:
+            with client.session_transaction() as session:
                 session['google_oauth2_csrf_token'] = 'tokenz'
 
             state = '[{'
-            rv = c.get('/oauth2callback?state=%s&code=codez' % state)
-            self.assertEqual(rv.status_code, httplib.BAD_REQUEST)
+            response = client.get('/oauth2callback?state=%s&code=codez' % state)
```

Comment:
Any reason for using `.format` in some places in this comment and `%` in other places?